app/models/stacks_file.rb

```diff
@@ -15,12 +15,6 @@ def file_exist?
     path && File.exist?(path)
   end
 
-  def valid?
-    # FIXME: image_exist? is defined in app/models/concerns/djatoka_adapter.rb,
-    #  which is included in StacksImage, but is probably undefined here
-    image_exist?
-  end
```

Comment:
yeah - those are good suggestions for when we are trying to do something useful with valid?